core/src/App.html

```diff
@@ -641,7 +670,9 @@
                 }
                 resolve();
               },
-              () => {}
+              e => {
+                console.error('E', e);
```

Comment:
In case we want to keep the `console.error`, can we say "Error in handleNavigation" or sth similar instead of "E"?

Reply:
was just a test, guess we don't need it